htdocs/core/actions_massactions.inc.php

```diff
@@ -728,13 +728,14 @@
 	}
 
 	$arrayofinclusion=array();
-	foreach($listofobjectref as $tmppdf) $arrayofinclusion[]='^'.preg_quote($tmppdf.'.pdf','/').'$';
+	foreach($listofobjectref as $tmppdf) $arrayofinclusion[]='^'.preg_quote(preg_replace('#/#','_',$tmppdf).'.pdf','/').'$';
```

Comment:
What do you think of using 
dol_sanitizeFileName instead of the preg_replace to replace / ?